README.md

```diff
@@ -95,6 +95,7 @@ module "ipa1" {
 | nessus\_key\_key | The SSM Parameter Store key whose corresponding value contains the secret key that the Nessus Agent should use when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/key). | `string` | n/a | yes |
 | nessus\_port\_key | The SSM Parameter Store key whose corresponding value contains the port to which the Nessus Agent should connect when linking with the CDM Tenable Nessus server (e.g. /cdm/nessus/port). | `string` | n/a | yes |
 | realm | The realm for the IPA server (e.g. EXAMPLE.COM).  Only used if this IPA server IS NOT intended to be a replica. | `string` | `"EXAMPLE.COM"` | no |
+| root\_disk\_size | The size of the IPA instance's root disk in GB. | `number` | `8` | no |
```

Comment:
This is getting into really pedantic turf, but it's supported [in the documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#volume_size).

```suggestion
| root\_disk\_size | The size of the IPA instance's root disk in GiB. | `number` | `8` | no |
```